build/gulpfile.vscode.web.js

```diff
@@ -32,8 +32,10 @@ const WEB_FOLDER = path.join(REPO_ROOT, 'remote', 'web');
 
 const commit = getVersion(REPO_ROOT);
 const quality = product.quality;
+const version = (quality && quality !== 'stable') ? `${packageJson.version}-${quality}` : packageJson.version;
```

Comment:
Does this need a Positron code fence?

Reply:
This line has been restored to the original from upstream https://github.com/microsoft/vscode/blob/7473068e0d7b62bfd4ef4c8f6b86832448ff1d36/build/gulpfile.vscode.web.js#L32, so I think we're good!

But I did miss a fence in the `build/gulpfile.reh.js` file so thank you for bringing this up 🙌